opensource_git_contrib/README.md

```diff
@@ -22,4 +22,4 @@ Resources: [Link](https://codeburst.io/a-step-by-step-guide-to-making-your-first
 4. Thats it!
 
 This is an oversimplified way of contributing
-For more advanced usage we can use branches and commands like adding upstream url and doing `git fetch`. Please read online about these concepts.
\ No newline at end of file
+For more advanced usage we can use branches and commands like adding upstream url and doing `git fetch`. Please read online about these concepts. A great Explaination can be found [HERE](https://gist.github.com/Chaser324/ce0505fbed06b947d962). It's a must read for everyone.
```

Comment:
'A great explanation can be found [here](......)'. Correct this sentence. Rest is fine.